src/MyTested.AspNetCore.Mvc.EntityFrameworkCore/Builders/Data/WithoutDbContextBuilder.cs

```diff
@@ -0,0 +1,102 @@
+﻿namespace MyTested.AspNetCore.Mvc.Builders.Data
+{
+    using Microsoft.EntityFrameworkCore;
+    using MyTested.AspNetCore.Mvc.Builders.Base;
+    using MyTested.AspNetCore.Mvc.Builders.Contracts.Data;
+    using MyTested.AspNetCore.Mvc.Internal.TestContexts;
+    using MyTested.AspNetCore.Mvc.Utilities.Validators;
+    using System;
```

Comment:
`System` usings should be on top and `MyTested` usings should use the short syntax. Instead of `MyTested.AspNetCore.Mvc.Builders.Base` it should be only `Base`, for example.